tests/static_temperature/statistics

```diff
@@ -0,0 +1,18 @@
+# 1: Time step number
+# 2: Time (years)
+# 3: Time step size (years)
+# 4: Number of mesh cells
+# 5: Number of Stokes degrees of freedom
+# 6: Number of temperature degrees of freedom
+# 7: Iterations for temperature solver
+# 8: Iterations for Stokes solver
+# 9: Velocity iterations in Stokes preconditioner
+# 10: Schur complement iterations in Stokes preconditioner
+# 11: RMS velocity (m/year)
+# 12: Max. velocity (m/year)
+# 13: Minimal temperature (K)
+# 14: Average temperature (K)
+# 15: Maximal temperature (K)
+# 16: Visualization file name
+0 0.000000000000e+00 0.000000000000e+00 256 2467 1089 0 32 34 33 1.48810793e-01 4.41088699e-01  0.00000000e+00 1.34592014e+03 1.40000000e+03 static_temperature/solution/solution-00000 
+1 2.000000000000e+04 2.000000000000e+04 256 2467 1089 9 28 30 30 1.47737066e-01 4.35559826e-01 -1.00484178e+02 1.34593415e+03 1.45657670e+03 static_temperature/solution/solution-00001 
```

Comment:
This can't be quite correct yet (maybe because you haven't actually set the temperature field to 'static' yet): The temperature statistics in the last few columns still changes :-)